Mathlib/Data/Set/Image.lean

```diff
@@ -720,6 +730,10 @@ theorem image_univ {f : α → β} : f '' univ = range f := by
   simp [image, range]
 #align set.image_univ Set.image_univ
 
+@[simp]
+theorem univ_subset_preimage {f : α → β} (s) : univ ⊆ f ⁻¹' s ↔ range f ⊆ s := by
```

Comment:
I think you should make `univ_subset_iff` simp instead

Reply:
I would still need a simp lemma `f ⁻¹' s = univ ↔ range f ⊆ s` as well, right? Would you suggest replacing this lemma with one about `= univ` and making `univ_subset_iff` simp?

Reply:
Yes, exactly. I believe we independently want `f ⁻¹' s = univ ↔ range f ⊆ s` as a simp lemma anyway.